pinot-common/src/main/java/org/apache/pinot/common/protocols/SegmentCompletionProtocol.java

```diff
@@ -201,6 +205,7 @@ public String getUrl(String hostPort, String protocol) {
       private String _segmentLocation;
       private long _memoryUsedBytes;
       private long _segmentSizeBytes;
+      private boolean _segmentUploadToController;
```

Comment:
nit: rename to "_uploadToControllerEnabled" ?

Reply:
done.